src/connectivity.ts

```
@@ -0,0 +1,267 @@
+      if (aiSettings.textModels.length > 0) {
+        text += "### 💬 Text Models\n\n";
+        aiSettings.textModels.forEach(m => text += `* ${m.name}\n`);
+        text += "\n";
+      } else {
+        text += "### 💬 Text Models\n\n_No text models configured._\n\n";
+      }
+
+      if (aiSettings.imageModels.length > 0) {
+        text += "### 🎨 Image Models\n\n";
+        aiSettings.imageModels.forEach(m => text += `* ${m.name}\n`);
+        text += "\n";
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)